metricbeat/tests/system/test_autodiscover.py

```diff
@@ -80,18 +83,73 @@ def test_docker_labels(self):
             'co.elastic.metrics/period': '1s',
             'co.elastic.metrics/hosts': "'${data.host}:11211'",
         }
-        container = docker_client.containers.run('memcached:1.5.3', labels=labels, detach=True)
+        container = docker_client.containers.run(
+            'memcached:1.5.3', labels=labels, detach=True)
 
-        self.wait_until(lambda: self.log_contains('Starting runner: memcached'))
+        self.wait_until(lambda: self.log_contains(
+            'Starting runner: memcached'))
 
         self.wait_until(lambda: self.output_count(lambda x: x >= 1))
         container.stop()
 
-        self.wait_until(lambda: self.log_contains('Stopping runner: memcached'))
+        self.wait_until(lambda: self.log_contains(
+            'Stopping runner: memcached'))
 
         output = self.read_output_json()
         proc.check_kill_and_wait()
 
         # Check metadata is added
         assert output[0]['docker']['container']['image'] == 'memcached:1.5.3'
         assert 'name' in output[0]['docker']['container']
+
+    @unittest.skipIf(not INTEGRATION_TESTS)
```

Comment:
We normally use `@unittest.skipUnless(INTEGRATION_TESTS, "integration test")` I think CI doesn't like this variant.